pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -15,8 +21,23 @@ buildGoModule rec {
 
   doCheck = false;
 
+  nativeBuildInputs = [
+    debianutils
+    mkdocs
+    nodejs
+    python3
+    python3Packages.mkdocs-material
+    python3Packages.mkdocs-minify
+  ];
+
+  postPatch = ''
+    sed -i 's# /bin/echo# echo#' Makefile
+    sed -i 's/version = "dev"/version = "${version}"/' main.go
```

Comment:
Setting `ldflags = [ "-s" "-w" "-X main.version=${version}" ]` is the cleaner thing to do here (and is what the [official build does](https://github.com/binwiederhier/ntfy/blob/46fcbdb8277a93224f4b1cc9e58c1f536cda02da/.goreleaser.yml#L12-13)).

(The first two flags strip the binary, which we'll eventually [do by default](https://github.com/NixOS/nixpkgs/issues/177698), but until then, let's set them here.)